script/start_qp.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+if [ ! -f "$(dirname "$(dirname "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )" )" )/static/.env.production.local" ]; then
+    echo "Missing updated .env.production.local file in the static directory. The server may not function properly"
+else
+    cp "$(dirname "$(dirname "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )" )" )/static/.env.production.local" "$(dirname "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )" )"
+fi
+
+export RAILS_RELATIVE_URL_ROOT=/aaec
+export RAILS_ENV=production
+
+bundle exec rake db:seed
+
+# Check if puma is started
+if ! ( [ -f $APP_DIR/tmp/puma/pid ] && pgrep -F $APP_DIR/tmp/puma/pid > /dev/null )
```

Comment:
Your conditional is always returning `true` because `$APP_DIR` isn't defined.  It should be set to `/opt/webapps/aaec/current`